packages/notebook-viewer/Dockerfile

```diff
@@ -1,15 +1,16 @@
 ARG REGISTRY
-FROM ${REGISTRY}python:3.9.1
+FROM ${REGISTRY}python:3.13.1-alpine
 
 RUN mkdir -p /usr/src/app
-RUN apt-get update
+RUN apk update
+RUN apk upgrade
 
 WORKDIR /usr/src/app
 
 COPY . .
 
 RUN pip install -r requirements.txt
 
-CMD python -m flask run --host=0.0.0.0 --port 8080
+CMD ["gunicorn", "-b", "0.0.0.0:8080", "app:app"] 
```

Comment:
What's the reasoning behind running gunicorn for this?

Reply:
Flask is a development server, so in their server logs they output something like "this is a development server; don't run it in production". [In their docs](https://flask.palletsprojects.com/en/stable/deploying/), they recommend a number of WSGI servers to choose from. `gunicorn` is one that I've used before, so I picked it based on familiarity.